src/auth/screens/auth-profile.screen.js

```diff
@@ -36,6 +37,9 @@ const styles = StyleSheet.create({
     color: colors.greyDark,
     ...fonts.fontPrimary,
   },
+  logoutTitle: {
+    color: colors.red,
```

Comment:
The primary font, `...fonts.fontPrimary` should also be applied here as well! :)